github-projects@morgan-design.com/files/github-projects@morgan-design.com/po/github-projects.pot

```diff
@@ -1,209 +1,436 @@
-# SOME DESCRIPTIVE TITLE.
+# Swedish translation for github-projects@morgan-design.com.
```

Comment:
Shouldn't this be in an sv.po file? This is the template pot file.

Reply:
Sorry, I don't understand. It is sv.po as far as I can see.